File: Temp.js
function twoSum(arr, target) {return true}

const fs = require('fs');
const input = fs.readFileSync('input.txt', 'utf8').trim().split('\n');
const output = fs.readFileSync('output.txt', 'utf8').trim().split('\n');
let result = '';

let t = parseInt(input[0]);
let line = 1;

while (t--) {
  const arr = input[line++].split(',').map(Number);
  const target = parseInt(input[line++]);

  const ans = twoSum(arr, target);
  const expected = output.shift().split(',').map(Number);

  if (JSON.stringify(ans) === JSON.stringify(expected)) {
    result += 'Output matched\n';
  } else {
    result += 'Wrong answer\n';
  }
}

fs.writeFileSync('result.txt', result);

File: Temp.java
import java.io.*;
import java.util.*;

public class Main {
    public static int[] twoSum(int[] arr, int target) { return new int[]{-1};}

    public static void main(String[] args) throws Exception {
        BufferedReader input = new BufferedReader(new FileReader("input.txt"));
        BufferedReader output = new BufferedReader(new FileReader("output.txt"));
        BufferedWriter result = new BufferedWriter(new FileWriter("result.txt"));

        int t = Integer.parseInt(input.readLine());

        while (t-- > 0) {
            String[] arrStr = input.readLine().split(",");
            int[] arr = Arrays.stream(arrStr).mapToInt(Integer::parseInt).toArray();
            int target = Integer.parseInt(input.readLine());

            int[] ans = twoSum(arr, target);
            String[] outStr = output.readLine().split(",");
            int[] expected = Arrays.stream(outStr).mapToInt(Integer::parseInt).toArray();

            if (Arrays.equals(ans, expected)) {
                result.write("Output matched\n");
            } else {
                result.write("Wrong answer\n");
            }
        }

        input.close();
        output.close();
        result.close();
    }
}